guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,478 @@
 {
```

Comment:
<ol style="user-select: auto;"><li style="user-select: auto;"><code style="user-select: auto;"><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.analytics.html#manage-multidimensional-raster" target="_blank" style="user-select: auto;">manage_multidimensional_raster </a></code>is listed twice</li><li>Bullet point and code formatting for "costpath_as_polyline" needs to be checked</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Removed the extra <a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.analytics.html#manage-multidimensional-raster" target="_blank">manage_multidimensional_raster </a> and the bullet point/indentation for costpath_as_polyline</p>